ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,25 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def check_ceph_health_after_add_capacity():
```

Comment:
make sure you use ceph_health_tries as a parameter data we might need more in some cases

Reply:
Okay, I will add it to the function.